quotefault/mail.py

```diff
@@ -1,15 +1,39 @@
-import smtplib
-import os
+"""
+File name: mail.py
+Author: Nicholas Mercadante
+Contributors: Joe Abbate
+"""
 from email.mime.text import MIMEText
 from email.mime.multipart import MIMEMultipart
+import smtplib
+from flask import render_template
+from flask_mail import Mail, Message
+from quotefault import app
+
+mail_client = Mail(app)
 
-def send_email(app, toaddr, subject, body):
-    fromaddr = "quotefault@csh.rit.edu"
+def send_report_email(reporter, quote):
+    """
+    Send email to eboard/rtp for a new report
+    """
+    recipients = ["<eboard@csh.rit.edu>","<rtp@csh.rit.edu>"]
+    msg = Message(subject='New QuoteFault Report',
+                sender=app.config.get('MAIL_USERNAME'),
+                recipients=recipients)
+    template = 'mail/report'
+    msg.body = render_template(template + '.txt', reporter = reporter, quote = quote )
+    msg.html = render_template(template + '.html', reporter = reporter, quote = quote )
+    mail_client.send(msg)
+
+def send_email(toaddr, subject, body):
+    """
+    Helper function for quote notification email
+    """
+    fromaddr = app.config['MAIL_USERNAME']
     msg = MIMEMultipart()
     msg['From'] = fromaddr
     msg['To'] = toaddr
     msg['Subject'] = subject
-    body = body
```

Comment:
Why is this deleted?

Reply:
It's body = body, it did nothing

Reply:
Is it supposed to be, like, `msg['Body'] = body` or something?

Reply:
body is passed as its own parameter